docs/sources/articles/remote_api_authorization.md

```diff
@@ -0,0 +1,103 @@
+# Docker's remote API authorization with macaroons
+
+Docker uses [Macaroons](http://research.google.com/pubs/pub41892.html) for authorize requests to the remote API. This article explains how to grant privileges for a remote client to interact with the API.
+
+## Initializing the credentials in the server
+
+The first command to run is `docker id --init`. This creates the root macaroon in the server and it will be shared with the client.
+
+```
+$ docker id --init --secret "Super secret token"
+```
+
+From that point forward, any request to the api that doesn't include the macaroon will be unauthorized.
+
+The macaroon is stored in the client in its serialized format. It can look like this:
+
+```
+eyJjYXZlYXRzIjpbXSwibG9jYXRpb24iOiJ0aGUgY2xvdWQiLCJpZGVudGlmaWVyIjoiZG9ja2VyIiwic2lnbmF0dXJlIjoiNzE4ZGJkZTQwZDliZGFhNjM4MTUzOTQxZTA3MjY4YjlhYTkzYzk1ODIyMzVlMGI4ZWM0NDgzZGQ4ZmVkNTMyZiJ9
+```
+
+Deserializing the macaroon we can see more details about the current authorization credentials:
+
+```json
+{
+  "caveats":[],
+  "location":"the cloud",
+  "identifier":"docker",
+  "signature":"718dbde40d9bdaa638153941e07268b9aa93c9582235e0b8ec4483dd8fed532f"
+}
+```
+
+Any client in possession of this macaroon will have full read/write access, since we're not including any caveat.
+
+## Sharing credentials
+
+You can always see a serialized version of your macaroon by running `docker id --show`.
+
+```
+$ docker id --show
+eyJjYXZlYXRzIjpbXSwibG9jYXRpb24iOiJ0aGUgY2xvdWQiLCJpZGVudGlmaWVyIjoiZG9ja2VyIiwic2lnbmF0dXJlIjoiNzE4ZGJkZTQwZDliZGFhNjM4MTUzOTQxZTA3MjY4YjlhYTkzYzk1ODIyMzVlMGI4ZWM0NDgzZGQ4ZmVkNTMyZiJ9
+```
+
+You can share your serialized macaroon with any other client that needs access to the docker api. There are two ways to tell the docker client to use a specific macaroon.
```

Comment:
Could you provide an example of how sharing this macaroon with another client would work? Is this expected to be an out-of-band operation? Or integrated sharing through another mechanism?